tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -393,3 +393,41 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     );
   })
 };
+
+if (!/\/2\.8/.test(Cypress.env('rancher_version')) && !/\/2\.9/.test(Cypress.env('rancher_version'))) {
+  describe('Test GitJob security context',  { tags: '@p0' }, () => {
+    qase(160,
+      it('FLEET-160: Test GitJob pod security context', { tags: '@fleet-160' }, () => {
+        // Check the GitJob pod for Security Context.
+        cy.accesMenuSelection('local', 'Workloads', 'Pods');
+        cy.filterInSearchBox('gitjob');
+        cy.verifyTableRow(0, 'Running', 'gitjob');
+        cy.contains('gitjob').click();
+        cy.clickButton('Config');
+        cy.get('section#container-0')
+          .find('.side-tabs ul.tabs li')
+          .eq(3)
+          .should('have.id', 'securityContext')
```

Comment:
Suggestion: I'd add also a contains text validation in case the id is moved at some point to another element in the future

Reply:
Okay, I will add text validation.

Reply:
Thanks

Reply:
Thanks for checking. Approving